server/plugin/client.go

```diff
@@ -267,11 +268,46 @@ func (c *client) UpdatePipelineApprovalRequest(pipelineApproveRequestPayload *se
 	baseURL := c.plugin.getConfiguration().AzureDevopsAPIBaseURL
 	baseURL = strings.Replace(baseURL, "://", "://vsrm.", 1)
 	_, statusCode, err := c.CallJSON(baseURL, pipelineApproveRequestURL, http.MethodPatch, mattermostUserID, &pipelineApproveRequestPayload, nil, nil)
+
+	return statusCode, err
+}
+
+func (c *client) GetSubscriptionFilterPossibleValues(request *serializers.GetSubscriptionFilterPossibleValuesRequestPayload, mattermostUserID string) (*serializers.SubscriptionFilterPossibleValuesResponseFromClient, int, error) {
+	getSubscriptionFilterValuesURL := fmt.Sprintf(constants.GetSubscriptionFilterPossibleValues, request.Organization)
```

Comment:
can't we use this URL value directly instead of creating a new variable?

Reply:
we can, but to keep consistent and more readable I think we should keep it